extensions/wikia/Privacy/RemoveUserDataController.php

```diff
@@ -77,6 +86,17 @@ private function getUserWikis( int $userId ) {
 		return $specialsDbr->selectFieldValues( 'events_local_users', 'wiki_id', ['user_id' => $userId], __METHOD__, ['DISTINCT'] );
 	}
 
+	private function getFakeUser( $username ) {
```

Comment:
What does `fake` is this context mean?

Reply:
This is how the user, which is created during rename process, is named in the code:
https://github.com/Wikia/app/blob/25b19bc52b237f2c74d8d5a07d4d3e57c2228f37/extensions/wikia/UserRenameTool/RenameUserProcess.class.php#L443-L443

Reply:
I can add comment about it